src/schema-builder/RdbmsSchemaBuilder.ts

```diff
@@ -357,7 +357,20 @@ export class RdbmsSchemaBuilder implements SchemaBuilder {
 
             // drop all indices that exist in the table, but does not exist in the given composite indices
             const dropQueries = tableSchema.indices
-                .filter(indexSchema => !metadata.indices.find(indexMetadata => indexMetadata.name === indexSchema.name))
+                .filter(indexSchema => {
+                    var metadataIndex = metadata.indices.find(indexMetadata => indexMetadata.name === indexSchema.name);
```

Comment:
don't use `var` - use `const` instead. Also take a look, you have tslint errors

Reply:
Done